README.rst

```diff
@@ -13,7 +13,7 @@ border would share the same color.
   :align: center
   :figclass: align-center
   
-  A graph representation of Canada's provinces. Provinces connected by edges
+  A graph representation of Canada's provinces of Canada. Provinces connected by edges
```

Comment:
Typo. Perhaps `A graph representation of the provinces of Canada.`

Reply:
Thanks , updated the as per comment, please review